lib/manageiq/api/common/application_controller_mixins/parameters.rb

```diff
@@ -0,0 +1,95 @@
+module ManageIQ
+  module API
+    module Common
+      module ApplicationControllerMixins
+        module Parameters
+          def params_for_create
+            # We already validate this with OpenAPI validator, that validates every request, so we shouldn't do it again here.
+            body_params.permit!
```

Comment:
Now that you have openapi_enabled = false...it's possible we could now be permitting bad params to your non-openapi_enabled controllers... cc @bdunne

Reply:
This probably goes for almost all of the methods in here.

Reply:
@Fryguy But you would be calling these methods only from openapi enabled controllers. A non openapi enabled controller doesn't have to call these methods

Reply:
No, but it's not uncommon from a controller to call methods like `params_for_create`, which would now be available to them.  At a minimum maybe these methods should raise an Exception unless openapi_enabled?